modules/io/include/visp3/io/vpImageIo.h

```diff
@@ -153,10 +162,10 @@ class VISP_EXPORT vpImageIo
   static void writePPM(const vpImage<unsigned char> &I, const std::string &filename);
   static void writePPM(const vpImage<vpRGBa> &I, const std::string &filename);
 
-  static void writeJPEG(const vpImage<unsigned char> &I, const std::string &filename);
-  static void writeJPEG(const vpImage<vpRGBa> &I, const std::string &filename);
+  static void writeJPEG(const vpImage<unsigned char> &I, const std::string &filename, int quality=90, int backend=IO_DEFAULT_BACKEND);
+  static void writeJPEG(const vpImage<vpRGBa> &I, const std::string &filename, int quality=90, int backend=IO_DEFAULT_BACKEND);
 
-  static void writePNG(const vpImage<unsigned char> &I, const std::string &filename);
-  static void writePNG(const vpImage<vpRGBa> &I, const std::string &filename);
+  static void writePNG(const vpImage<unsigned char> &I, const std::string &filename, int backend=IO_DEFAULT_BACKEND);
+  static void writePNG(const vpImage<vpRGBa> &I, const std::string &filename, int backend=IO_DEFAULT_BACKEND);
```

Comment:
Currently only with `JPEG` we can select the quality, but theoretically we should do the same with PNG.

Better, something like a `vector` of options similar to OpenCV `ImwriteFlags`: https://docs.opencv.org/4.5.5/d8/d6a/group__imgcodecs__flags.html#ga292d81be8d76901bff7988d18d2b42ac

```cpp

```

Reply:
- PNG quality could be introduced in a future release.
- For `writeJPEG()`, I suggest to add first the backend, and then the quality

```
  static void writeJPEG(const vpImage<unsigned char> &I, const std::string &filename, int backend=IO_DEFAULT_BACKEND, int quality=90);
  static void writeJPEG(const vpImage<vpRGBa> &I, const std::string &filename, int backend=IO_DEFAULT_BACKEND, int quality=90);
```
so that if we add quality to `writePNG()` the prototype remain similar

Reply:
:+1:

Reply:
Resolved in commit 9180a98